gnovm/pkg/gnolang/nodes_string.go

```diff
@@ -13,7 +13,7 @@ var wordTokenStrings = map[Word]string{
 	SUB:             "-",
 	MUL:             "*",
 	QUO:             "/",
-	REM:             "%",
+	REM:             "%%",
```

Comment:
Was this done so that it renders correctly when passed to `fmt.Printf`?

Reply:
Yes, sometimes it's necessary to trigger errors tied to specific operators.

Reply:
In that case I think it would be useful to put a comment here explaining why the percent signs have been escaped. I was a bit unsure myself when I first saw it so maybe it would also be helpful for future readers of the code.

Reply:
this is handled on the caller side.